source/particle/property/elastic_stress.cc

```diff
@@ -74,56 +85,64 @@ namespace aspect
 
       }
 
+
+
       template <int dim>
       void
       ElasticStress<dim>::update_particle_property(const unsigned int data_position,
                                                    const Vector<double> &solution,
                                                    const std::vector<Tensor<1,dim> > &gradients,
                                                    typename ParticleHandler<dim>::particle_iterator &particle) const
       {
-        MaterialModel::MaterialModelInputs<dim> in(1, this->n_compositional_fields());
+        //material_inputs = MaterialModel::MaterialModelInputs<dim>(1, this->n_compositional_fields());
```

Comment:
remove the commented line

Reply:
@gassmoeller - done and same with the line below. I squashed the commit in the last one.